Sources/UID2GMAPlugin/UID2GMAMediationAdapter.swift

```diff
@@ -19,6 +19,14 @@ class UID2GMAMediationAdapter: NSObject {
 
 extension UID2GMAMediationAdapter: GADRTBAdapter {
 
+    static func setUpWith(_ configuration: GADMediationServerConfiguration, completionHandler: @escaping GADMediationAdapterSetUpCompletionBlock) {
```

Comment:
does the _ mean it's an optional param in swift?

Reply:
Nope.  It's a way to omit the need to use a parameter name when calling a function (see link below).  This feature is available to help make functions more human readable.

https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions/#Omitting-Argument-Labels

In this case, this means the following:

```swift
/// Correct
UID2GMAMediationAdapter.setUpWith(GADMediationServerConfiguration(), completionHandler: { error in
      // Error handling code goes here
})

/// Not Correct
UID2GMAMediationAdapter.setUpWith(configuration: GADMediationServerConfiguration(), completionHandler: { error in
      // Error handling code goes here
})

```

Reply:
ah i see! but i thought param labeling is the fundamental feature of obj-c/swift and labeling make it more readable (more typing of course)

Reply:
> make it more readable 

This is really the goal that Swift is chasing with the parameter labeling / omitting feature.  In some cases it makes more sense to use a label and in some other cases it doesn't.  For example:

```swift
func updateFoo(_ foo: Foo) { }

becomes

let foo = Foo()
updateFoo(foo)

OR

func doSomethingMagical(foo: Foo) { }
let foo = Foo()
doSomethingMangical(foo: foo)

OR

func doSomethingMagical(with foo: Foo) { }

becomes

let foo = Foo()
doSomethingMangical(with: foo)

```